content/docs/user-guide/best-practices.md

```diff
@@ -0,0 +1,133 @@
+# Best Practices for DVC Projects
+
+DVC provides a systematic approach towards managing and collaborating on data
+science projects. Here are a few recommended practices to organize your workflow
+and project structure effectively:
+
+> See also these quick [tips & tricks](/doc/user-guide/tips-and-tricks).
+
+## Matching source code to data
+
+One of DVC's basic uses is to avoid a disconnection between
+[revisions](https://git-scm.com/docs/revisions) of source code and
+[versions](/doc/use-cases/versioning-data-and-model-files) of data. DVC replaces
+large data files and directories, models, etc. with small
```

Comment:
This isn't really a best practice, just an intro to data tracking with DVC. I guess it could stay here... Not sure 🤔